integration-cli/docker_cli_cp_utils_test.go

```diff
@@ -103,8 +108,8 @@ func makeTestContentInDir(c *check.C, dir string) {
 		case ftSymlink:
 			c.Assert(os.Symlink(fd.contents, path), checker.IsNil)
 		}
-
-		if fd.filetype != ftSymlink && runtime.GOOS != "windows" {
+		// Need skip Hardlink
```

Comment:
I don't think this comment provides any value over reading the code. It's actually confusing because the code skips both kinds of links, but the comment only talks about hard links.